config/settings.yml

```diff
@@ -1045,6 +1045,7 @@ lighthouse:
     use_mocks: false
   veterans_health:
     url: https://sandbox-api.va.gov
+    use_mocks: false
```

Comment:
This will dictate the local settings, but this value is also set in dev, prod and staging. Staging, for example:
https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/main/apps/vets-api/staging/values.yaml#L2183

The parameter store path: `/dsva-vagov/vets-api/staging/lighthouse_veterans_health_use_mocks`

Maybe you already knew this, but I just wanted to clarify!

Reply:
yep, thanks! Mainly just adding as a reference for folks to know the param exists :-)